lib/stacks/eks-blueprint-stack.ts

```diff
@@ -281,4 +289,47 @@ export class EksBlueprint extends cdk.Stack {
             });
         }
     }
-}
\ No newline at end of file
+}
+function generalConstraintsValidation(blueprintProps: EksBlueprintProps) {
```

Comment:
Same feedback on the naming: verb, general => input

Reply:
Fixed now.